gr-digital/grc/digital_constellation_soft_decoder_cf.block.yml

```diff
@@ -6,6 +6,10 @@ parameters:
 -   id: constellation
     label: Constellation Object
     dtype: raw
+-   id: npwr
+    label: Noise Power
+    dtype: float
+    default: '-1'
```

Comment:
Is a default noise power of `-1` intended?

Reply:
Yes, this is to allow the value set in the constellation object to take precedence. I.e. if you specified a LUT calculated for some sigma/SNR and you don't want it to be overriden by the decoder block.

Reply:
@daniestevez just saw your edit, I agree that it's surprising, but this defaults the encoder to not override the noise power set by the constellation object. I feel this is a rational default since it lets the user know that's an option that they may otherwise never know. This allows a LUT to be set without being overridden by a change in noise power for instance. Should probably just add some additional documentation to the block to explain this.